src/streaming/zarr.stream.cpp

```diff
@@ -7,13 +7,18 @@
 #include "sink.hh"
 
 #include <blosc.h>
+#include <omp.h>
 
 #include <filesystem>
 
 #ifdef min
 #undef min
 #endif
 
+#ifdef max
```

Comment:
You might be able to just say #undef without needing the #ifdef.

More importantly, where are these symbols leaking in from? If possible you should handle this as close to the source as possible.